### DISCUSSION TITLE
go.d/ping organisation of graphics

### DISCUSSION THREAD

**Discussion by cayenne17** [netdata/netdata#14514] [Q&A]
Hello,
I just learned that the fping mobule is deprecated since version 1.37 of netdata : https://github.com/netdata/netdata/issues/14508

I am testing and configuring go.d/ping https://github.com/netdata/go.d.plugin/tree/master/modules/ping

I would like to have a similar organization of the graphs to fping with go.d/ping because it forces to scroll to check the ping/loss of packet. 
i.e. : 

- host_1 Lantency
- host_1 packets
- host_1 packet loss
- host_2 Lantency
- host_2 packets
- host_2 packet loss
- host_3 Lantency
- host_3 packets
- host_3 packet loss

It used to be like this for a host fping :
![20230202_22h45_48s_vivaldi_0SL83yAE6l](https://user-images.githubusercontent.com/47927025/218437081-cd8c40ff-eb27-4c4a-81d6-8328c0acc95b.png)

Now, with go.d/ping : 

- host_1 Lantency
- host_2 Lantency
- host_3 Lantency
- host_1 packets
- host_2 packets
- host_3 packets
- host_1 packet loss
- host_2 packet loss
- host_3 packet loss

![image](https://user-images.githubusercontent.com/47927025/218439197-daa68edf-47a2-4689-b788-e356874add15.png)

**Comment by ilyam8** [comment]
Hi, @cayenne17. You are correct, the chart organization has been changed. We did this so that charts can be aggregated on Netdata Cloud.

You get aggregated charts, and per-host data can be seen by changing "group by" to "host":

<img width="1253" alt="Screenshot 2023-02-13 at 13 49 11" src="https://user-images.githubusercontent.com/22274335/218450224-e4db0bff-ccf2-46a5-81d5-d2e9984a5528.png">

There are no plans to get back to the previous charts organization.
